tests/components/ituran/conftest.py

```diff
@@ -0,0 +1,69 @@
+"""Mocks for the Ituran integration."""
+
+from datetime import datetime
+
+from pyituran import Ituran
+import pytest
+
+
+class MockVehicle:
```

Comment:
I think if you just do `self.make = "something"` that's more concise. But don't you just have the object from the library you can maybe autospec